hack/observability/cluster-api-visualizer/fetch-visualizer-resources.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CHART_RELEASE=${CHART_RELEASE:-0.1.0}
+VISUALIZER_ROOT=$(dirname "${BASH_SOURCE[0]}")
+CHART_ROOT=$VISUALIZER_ROOT/chart
+
+rm -rf "${CHART_ROOT:?}/"*
+wget -qO- https://raw.githubusercontent.com/Jont828/cluster-api-visualizer/main/helm/repo/cluster-api-visualizer-"$CHART_RELEASE".tgz \
```

Comment:
does this need to be changed according to #2306 ?

Reply:
Yeah it should. I had an error when running that PR so once I chat with Matt about it I'll update this to reflect those changes.